web-marketplace/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -93,9 +94,23 @@ const Dashboard = (): JSX.Element => {
   return (
     <>
       <ProfileHeader
-        name={party?.name ? party?.name : DEFAULT_NAME}
-        backgroundImage={party?.bgImage ? party?.bgImage : DEFAULT_PROFILE_BG}
-        avatar={party?.image ? party?.image : defaultAvatar}
+        name={party?.name ? party.name : DEFAULT_NAME}
+        backgroundImage={
+          party?.bgImage
+            ? getResizedImageUrl({
+                url: party?.bgImage,
+                width: 1400,
+              })
+            : DEFAULT_PROFILE_BG
+        }
+        avatar={
+          party?.image
+            ? getResizedImageUrl({
+                url: party?.image,
+                width: 140,
```

Comment:
why using 140? it now looks a bit blurry on https://deploy-preview-2106--regen-marketplace.netlify.app/profiles/regen1df675r9vnf7pdedn4sf26svdsem3ugavgxmy46/portfolio